site/en/docs/web-platform/view-transitions/index.md

```diff
@@ -104,34 +101,37 @@ And just like that, pages cross-fade:
 
 Ok, a cross-fade isn't that impressive. Thankfully, transitions can be customized, but before we get to that, we need to understand how this basic cross-fade worked.
 
+{% Aside %}
+Demo links in this article are less visually impressive, but also have a tiny codebase. The goal is to make it easier to read and understand all of the code, without knowing any particular SPA framework.
+{% endAside %}
+
 ## How these transitions work
 
 Taking the code sample from above:
 
 ```js
-const transition = document.createDocumentTransition();
-transition.start(() => updateTheDOMSomehow(data));
+document.startViewTransition(() => updateTheDOMSomehow(data));
 ```
 
-When `.start()` is called, the API captures the current state of the page. This includes taking a screenshot.
+When `.startViewTransition()` is called, the API captures the current state of the page. This includes taking a screenshot.
 
-Once that's complete, the callback passed to `.start()` is called. That's where the DOM is changed. Then, the API captures the new state of the page.
+Once that's complete, the callback passed to `.startViewTransition()` is called. That's where the DOM is changed. Then, the API captures the new state of the page.
 
 Once the state is captured, the API constructs a pseudo-element tree like this:
 
 ```diff
-::page-transition
-└─ ::page-transition-container(root)
-   └─ ::page-transition-image-wrapper(root)
-      ├─ ::page-transition-outgoing-image(root)
-      └─ ::page-transition-incoming-image(root)
+::view-transition
+└─ ::view-transition-group(root)
+   └─ ::view-transition-image-set(root)
+      ├─ ::view-transition-old(root)
+      └─ ::view-transition-new(root)
 ```
 
-The `::page-transition` sits in a top-layer, over everything else on the page.
+The `::view-transition` sits in a top-layer, over everything else on the page.
```

Comment:
```suggestion
The `::view-transition` sits in an overlay, over everything else on the page.
```

Technically not the "top layer" used by fullscreen or dialog. But IIUC you're just hitting at the fact thats its conceptually an overlay over everything else on the page